content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -79,7 +79,35 @@ $ dvc exp run --all-pipelines
 > Note that the order in which pipelines are executed is not guaranteed; Only
 > the internal order of stage execution is.
 
-### Interactive reproduction
+### Running pipelines recursively
+
+DVC supports pipelines defined in more than one `dvc.yaml` file. These can
+reside in nested folders and you may want to run all these in a single command.
+
+```
+$ tree
```

Comment:
````suggestion
```dvc
````

May or may not need `dvc` here, does it make a difference visually?

Reply:
I didn't try but I don't think so. We may better put `bash` if we remove `dvc` though, it may have some syntax coloring.

Reply:
`dvc` extends from `bash` or some other terminal highlighting I believe, which is why we don't use any other one. Please always render contributions to check how they look. Thanks

Reply:
`dvc` is needed - it expands bash, also It handles things like `$` (it becomes non-active text - no way to copy it), etc.

Reply:
Yeo but in the suggestion above I removed the `$ tree` line 😬 which is why I'm not sure. I'll merge it and see how it renders... ⏳

Reply:
OK so it doesn't seem to matter if there's no `$ tree` but I say we keep it since it doesn't hurt and the md code looks more consistent.

![image](https://user-images.githubusercontent.com/1477535/135203343-3744c70e-56d2-4675-aec9-75741f60c272.png)
https://dvc-org-iesahin-issue27-pwlhbh.herokuapp.com/doc/user-guide/experiment-management/running-experiments

> Unless you have a reason to keep `$ tree` @iesahin — keeping in mind it's Linux-specific (not a Windows tool and would technically be `tree -A` in Mac (close though).

Reply:
I didn't get the problem with `tree`. We're using it all over the documentation, even supposedly hands-on parts. We can skip it or show only a directory hierarchy but I don't think that will matter much.

Reply:
`tree` is GNU-specific. Ideally we should remove it from other places too IMO. The figure with the directory structure (`tree` output) is self explanatory I think. But yes this is minor so up to you.